data/src/main/java/org/oppia/android/data/backends/gae/model/GaeUserSuppliedFeedback.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for feedback reporting form model
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeUserSuppliedFeedback(
+
+  @Json(name = "report_type") val reportType: String?,
+  @Json(name = "category") val category: String?,
+  @Json(name = "feedback_list") val feedbackList: List<String>?,
+  @Json(name = "user_input") val userInput: String?
```

Comment:
Maybe: ``open_form_user_input`` might be a bit more explicit since all of this is technically user input. WDYT?

Reply:
Done, changed to `open_text_user_input` so it's explicit that it's a text response.